src/python/bot/tasks/task_creation.py

```diff
@@ -271,12 +273,19 @@ def request_bisection(testcase):
   if not target:
     return
 
-  _make_bisection_request(pubsub_topic, testcase, target, 'regressed')
-  _make_bisection_request(pubsub_topic, testcase, target, 'fixed')
+  # Only make 1 request of each type per testcase.
```

Comment:
I think you should be refetch testcase to avoid updating a stale one.
testcase = data_handler.get_testcase_by_id(testcase.key.id())

Reply:
Done!